client/src/App.tsx

```diff
@@ -210,6 +216,90 @@ function App() {
 		}
 	}, []);
 
+	const activateFireworks = () => {
+		var duration = 2 * 1000;
+		var animationEnd = Date.now() + duration;
+		var defaults = { startVelocity: 30, spread: 360, ticks: 60, zIndex: 0 };
+
+		function randomInRange(min: number, max: number) {
+			return Math.random() * (max - min) + min;
+		}
+
+		var interval: any = setInterval(function () {
+			var timeLeft = animationEnd - Date.now();
+
+			if (timeLeft <= 0) {
+				return clearInterval(interval);
+			}
+
+			var particleCount = 50 * (timeLeft / duration);
+			// since particles fall down, start a bit higher than random
+			confetti(
+				Object.assign({}, defaults, {
+					particleCount,
+					origin: { x: randomInRange(0.1, 0.3), y: Math.random() - 0.2 }
+				})
+			);
+			confetti(
+				Object.assign({}, defaults, {
+					particleCount,
+					origin: { x: randomInRange(0.7, 0.9), y: Math.random() - 0.2 }
+				})
+			);
+		}, 250);
+	};
+
+	const activateSnow = async () => {
+		var duration = 3.5 * 1000;
+		var animationEnd = Date.now() + duration;
+		var skew = 1;
+
+		function randomInRange(min: number, max: number) {
+			return Math.random() * (max - min) + min;
+		}
+
+		(function frame() {
+			var timeLeft = animationEnd - Date.now();
+			var ticks = Math.max(200, 500 * (timeLeft / duration));
+			skew = Math.max(0.8, skew - 0.001);
+
+			confetti({
+				particleCount: 1,
+				startVelocity: 0,
+				ticks: ticks,
+				gravity: 0.5,
+				origin: {
+					x: Math.random(),
+					// since particles fall down, skew start toward the top
+					y: Math.random() * skew - 0.2
+				},
+				colors: ['#ffffff'],
+				shapes: ['circle'],
+				scalar: randomInRange(0.4, 1)
+			});
+
+			if (timeLeft > 0) {
+				requestAnimationFrame(frame);
+			}
+		})();
+	};
+	const playMyAnimation = useCallback((animationType: string) => {
```

Comment:
Prefer a better name than playMyAnimation, I know playAnimation is taken so maybe we could change that to playTextAnimation and this function could be better named playAnimation